cockroachdb/example/cert-manager/kustomization.yaml

```diff
@@ -3,8 +3,10 @@ kind: Kustomization
 
 resources:
   - github.com/utilitywarehouse/shared-kustomize-bases//cockroachdb/manifests-cert-manager
-  - sa.yaml
   - db-init-manifests/example-database
+# sa.yaml and configmap cockroach.backup.config are resources necessary to add the backup.
+# If your use case excludes the usage of backups, use patch `disable-backup.yaml`
```

Comment:
This got me a bit confused. I think we need to mention that they need to also not include the sa.yaml, right?

Reply:
I modified the PR- now, clients doesn't have to store the "disable-backup" file and it's stored in the repository.
I also removed `kyverno-inject-sidecar`. 
I hope the documentation is more clear now. 
Pipeline fails, since the manifest-checker tries to render `Component` kustomization. I'll try to tackle it now.

Reply:
Nice, thank you!